src/components/organisms/modals/TooltipModal/TooltipModal.tsx

```diff
@@ -0,0 +1,69 @@
+import { FC, useCallback } from 'react'
+import { useTranslation } from 'react-i18next'
+import {
+  AppearanceTypes,
+  IconPositioningTypes,
+  SizeTypes,
+} from 'components/molecules/Button/Button'
+import ModalBase, {
+  ButtonPositionTypes,
+  TitleFontTypes,
+} from 'components/organisms/ModalBase/ModalBase'
+import classes from './styles.module.scss'
+
+export interface TooltipModalProps {
+  title?: string
+  textButtonContent?: string
+  modalContent?: string
+  href?: string
+  isModalOpen?: boolean
+  closeModal: () => void
+  onClose?: () => void
```

Comment:
Where do you use onClose func? I don't see that you pass on any func to onClose.